### DISCUSSION TITLE
Heuristic Protein inference_duplicate proteins instead of unique ID with quant values

### DISCUSSION THREAD

**Discussion by EddySomm** [vdemichev/DiaNN#913] [General]
Dear Vadim, Dear All
First of all apologies, probably the topic has been already discussed but I am struggling to find the solution
I have just started a test project on DIA NN, using default settings.
I checked the heuristic proiein inference box, but it seems that in the main report tsv files I do not have unique proteins, and furthermore ID protein name is still reported as as well as other files still contain duplicate "ProteinCenter:sp_canonicalXXXX" and not only with UniprotID. I have searched a unique protein ID matrix but I can see only report unique_genes_matrix, which is on the other hand correct since contains only genes and quant values
The main difficulty for me is that the data organization; I'm used to seeing something like the data output in Proteome Discoverer-- a list of proteins on the y axis and all the parameters such as intensity on the x axis.
Could you please help me?
Please find attached the log, and the screen of the tsv report (too big to upload)
![REPORT](https://github.com/vdemichev/DiaNN/assets/157610485/06c1be0f-8870-48e5-9361-cdd4a5734f6b)
[report.log.txt](https://github.com/vdemichev/DiaNN/files/14039660/report.log.txt)

Thank you in advance
Eduardo

**Comment by vdemichev** [comment]
Hi Eduardo,

Protein inference setting does not affect the 'Protein.Ids' column, the latter contains all protein IDs matched to the peptide - this is by design (inferred proteins are in Protein.Group). The IDs reported there are what is contained in the library or FASTA, can be any arbitrary sequence of characters.

The pg_matrix file should be in convenient format? 
It's not filtered for uniqueness though.
If you need this filter for protein sequence ids, not genes (in >95% cases it's better to consider uniqueness/proteotypicity with respect to genes, not isoform IDs), need to use R or Python to work with DIA-NN's main report.

Best,
Vadim
